smtp.py

```diff
@@ -23,7 +22,10 @@ async def send_mail(
 
 def valid_email(s):
     # https://regexr.com/2rhq7
-    pat = r"[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?"
+    pat = r"""
+    [a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@
+    (?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?
+    """
```

Comment:
is it OK to introduce new lines in regex?

Reply:
![screenshot-1722584364](https://github.com/user-attachments/assets/1067fb13-536b-48e8-ba8b-4a0e0b200e51)